docs/cloud_orchestrator.md

```diff
@@ -0,0 +1,17 @@
+# Cloud Orchestrator
+
+This page describes about `Cloud Orchestrator`.
+
+## What's Cloud Orchestrator?
+
+Cloud Orchestrator is a web service that managing virtual machines or containers
```

Comment:
nit: `for managing`

Reply:
https://github.com/google/cloud-android-orchestration/pull/321